doc/classes/Rect2.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Rect2" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		2D axis-aligned bounding box using floating point coordinates.
+		2D axis-aligned bounding box with floating-point coordinates.
 	</brief_description>
 	<description>
-		[Rect2] consists of a position, a size, and several utility functions. It is typically used for fast overlap tests.
-		It uses floating-point coordinates. If you need integer coordinates, use [Rect2i] instead.
-		The 3D counterpart to [Rect2] is [AABB].
-		Negative values for [member size] are not supported and will not work for most methods. Use [method abs] to get a Rect2 with a positive size.
+		The [Rect2] built-in [Variant] type represents a rectangle in 2D space. It is defined by its [member position] and [member size], which are [Vector2]. Because it does not rotate, it is frequently used for fast overlap tests (see [method intersects]).
+		For integer coordinates, use [Rect2i]. The 3D equivalent to [Rect2] is [AABB].
+		[b]Note:[/b] Negative values for [member size] are not supported and do work properly for most methods. Use [method abs] to get an equivalent [Rect2] with a positive size.
```

Comment:
Negative values for `size` do not work for a majority of methods because, quoting my correction in this PR, too:
> It's recommended setting the `size` to a positive value, as most methods in Godot assume that the `position` is the top-left corner, and the `end` is the bottom-right corner.

I do not believe the methods fail and return an error, however. They just are likely to return an incorrect or otherwise unexpected result. I could test this in more detail if necessary.

Reply:
I rewrote this note slightly. In hindsight, the original statement was rather vague, anyhow.